extensions/amp-analytics/0.1/requests.js

```diff
@@ -116,13 +117,24 @@ export class RequestHandler {
     const triggerParams = trigger['extraUrlParams'];
     const isImmediate =
         (trigger['immediate'] === true) || (this.maxDelay_ == 0);
+
+    const isV2ExpansionON = isExperimentOn(this.ampdoc.win,
```

Comment:
nit: since the experimental flag is used in different places, could you make it a const, and import from `url-replacements-impl.js` file.
nit: `isV2ExpansionOn` : )

Reply:
Since I think I need reference to the containing window, should this be a method on `GlobalVariableSource`?

Reply:
made const and fixed typo